### ISSUE TITLE
The GNN (APPNP) Branch is dead

### ISSUE THREAD

**Issue by johSchm** [ArdhenduBehera/SR-GNN#4]
The APPNP branch of the model seems to be a dead end at least in the provided code. Here the snippet from the `models.py`:
```
    N = num_rois + 1
    gcn_input=layers.Input(shape=(N, 2048 ) )

    print(' ')
    print('shape of gcn_input --> ', gcn_input.shape, type(gcn_input))
    print(' ')


    A = np.ones((N,N), dtype='int')
    fltr = GCNConv.preprocess(A).astype('f4')
    A_in = Input(tensor=sp_matrix_to_sp_tensor(fltr), name='AdjacencyMatrix')

    print(' ')
    print('shape of fltr --> ', fltr.shape)
    print('shape of A_in --> ', A_in.shape)
    print(' ')

    channels = 1024
    gc1 = APPNPConv(channels, alpha=alpha, propagations=1, mlp_activation='sigmoid', use_bias=True) ([gcn_input, A_in])

    print(' ')
    print('shape of gcn1 --> ', gc1.shape, type(gc1))
    print(' ')

    gc = APPNPConv(channels, alpha=alpha, propagations=1, mlp_activation='sigmoid', use_bias=True) ([gc1, A_in])
```
Both `APPNPConv` calls perform convolution on a complete graph (`A = np.ones((N,N), dtype='int')`) with placeholder inputs `gcn_input=layers.Input(shape=(N, 2048 ) )`. This essentially just creates a constant bias for the final logits vector before the class predication.

**Comment by ArdhenduBehera** [comment]
I believe you have found that the placeholder input is for creating the model. The actual call for the model with data input is in line #113.
